.travis.yml

```diff
@@ -2,12 +2,11 @@ language: dart
 
 dart:
   - dev
-  - 2.0.0
 
 dart_task:
-  - test: --platform vm
-  - test: --platform chrome
-  - dartanalyzer: --fatal-warnings --fatal-infos .
+  - test: --enable-experiment=non-nullable --platform vm
```

Comment:
I don't think this works - the dart_task test configuration doesn't allow vm arguments (this would be a test executable argument).

You actually don't need this though for this package (assuming it is in the allow list)

Reply:
> You actually don't need this though for this package (assuming it is in the allow list)

I thought we do need it so that the test files can be opted in?